### PR TITLE
[13.0][ENH] l10n_th_withholding_tax_cert, add create multi certs

### PR DESCRIPTION
**Pull request by kittiu**

### CODE REVIEW

File: l10n_th_withholding_tax_cert/tests/test_wt_cert.py [hunk 1]
```diff
@@ -156,26 +158,108 @@ def test_01_create_wt_cert_payment(self):
...
+        wizard = f.save()
+        wizard.write({"wt_account_ids": invoice.line_ids.mapped("account_id")})
+        res = wizard.create_wt_cert_multi()
+        certs = self.wt_cert.search(res["domain"])
+        self.assertEqual(len(certs), 2)
+
+    def test_04_create_wt_cert_je_multi(self):
+        """ Journal Entries to WT Certs """
+        misc_journal = self.browse_ref("account.miscellaneous_journal")
+        invoice = self._create_invoice(False, misc_journal.id, "entry", self.wt_account)
```
Comment (kittiu) [inline l10n_th_withholding_tax_cert/tests/test_wt_cert.py:235]: better name it "journal_entry" or "move", not "invoice" (this is not invoice, isn't it?)

File: l10n_th_withholding_tax_cert/tests/test_wt_cert.py [hunk 2]
```diff
@@ -156,26 +158,108 @@ def test_01_create_wt_cert_payment(self):
...
+        with Form(self.account_payment.with_context(ctx), view=view_id) as f:
+            f.journal_id = bank_journal
+            f.amount = 97.0  # To withhold 3.0
+            f.payment_difference_handling = "reconcile"
+            f.writeoff_account_id = self.wt_account
+            f.writeoff_label = "Withhold 3%"
+        payment = f.save()
+        payment.post()
+        self.assertEqual(payment.state, "posted")
+        payment2 = payment.copy()
```
Comment (kittiu) [inline l10n_th_withholding_tax_cert/tests/test_wt_cert.py]: When you copied the Payment, the WHT move line is not available.

image

You should create 2 invoices -> 2 payments. If the code is long, try to shorten by doing subrouting.

And make sure you test the

**Review by kittiu — COMMENTED** [review]
cc @ps-tubtim I will continue the fixing.

**Comment by kittiu** [comment]
cc @ps-tubtim , can you review/approve please.

**Review by ps-tubtim — APPROVED** [review]
LGTM

**Pull request merged**